src/accountsdb/db.zig

```diff
@@ -108,6 +110,8 @@ pub const AccountsDB = struct {
     /// NOTE: see accountsdb/readme.md for more details on how these are used
     file_map_fd_rw: std.Thread.RwLock,
 
```

Comment:
would be good to create a ticket to remove `maybe_accounts_cache_rw` -- so we have it tracked

Reply:
#537